packages/atomic/src/pages/index.html

```diff
@@ -206,6 +206,7 @@
           </atomic-layout-section>
           <atomic-layout-section section="results">
             <atomic-smart-snippet></atomic-smart-snippet>
+            <atomic-smart-snippet-suggestions></atomic-smart-snippet-suggestions>
```

Comment:
Eventually, that's likely to be moved between the first and second result in `index.html`. I'm likely going to introduce slots in `atomic-result-list` and `atomic-folded-result-list`.